minimal.rb

```diff
@@ -1,66 +1,53 @@
 run "if uname | grep -q 'Darwin'; then pgrep spring | xargs kill -9; fi"
 
-# GEMFILE
+# Gemfile
 ########################################
-inject_into_file 'Gemfile', before: 'group :development, :test do' do
+inject_into_file "Gemfile", before: "group :development, :test do" do
   <<~RUBY
-    gem 'autoprefixer-rails', '10.2.5'
-    gem 'font-awesome-sass'
-    gem 'simple_form', github: 'heartcombo/simple_form'
+    gem "autoprefixer-rails"
+    gem "font-awesome-sass", "~> 6.1"
+    gem "simple_form", github: "heartcombo/simple_form"
 
   RUBY
 end
 
-inject_into_file 'Gemfile', after: 'group :development, :test do' do
-  <<-RUBY
+inject_into_file "Gemfile", after: 'gem "debug", platforms: %i[ mri mingw x64_mingw ]' do
```

Comment:
```suggestion
inject_into_file "Gemfile", after: 'gem "debug", platforms: %i[mri mingw x64_mingw]' do
```

Reply:
Good catch :fishing_pole_and_fish: Thank you @Annedj :pray: